images/win/scripts/SoftwareReport/SoftwareReport.VisualStudio.psm1

```diff
@@ -88,4 +88,11 @@ function Get-VisualStudioExtensions {
     $extensions | Foreach-Object {
         [PSCustomObject]$_
     } | Select-Object Package, Version | Sort-Object Package
+}
+
+function Get-WindowsSDKs {
+    return [PSCustomObject]@{
+        path = "${env:ProgramFiles(x86)}\Windows Kits\10\Extension SDKs\WindowsDesktop"
+        versions = $(Get-ChildItem $path).Name
+    }
```

Comment:
looks like this doesn't work because you have to set `$path` before the PsCustomObject creation (That's why you receive multiple directories instead of one)
![image](https://user-images.githubusercontent.com/48208649/152577962-2713cfc2-7faf-4840-8d7d-d98582b832f0.png)